test/MSBuildBinLogQuery.CommandLine/MSBuildBinLogQuery.CommandLine.csproj

```diff
@@ -4,6 +4,7 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.0</TargetFramework>
     <RootNamespace>Microsoft.Build.Logging.Query.CommandLine</RootNamespace>
+    <LangVersion>preview</LangVersion>
```

Comment:
This should be default for `netcoreapp3.0`. Does it really need to be added?

Reply:
Done.

Reply:
As discussed, will keep it for now, as VS Code requires it.